WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteViewModel.kt

```diff
@@ -670,6 +676,7 @@ class MySiteViewModel @Inject constructor(
         domainRegistrationSource.clear()
         quickStartRepository.clear()
         scanAndBackupSource.clear()
+        selectedSiteSource.clear()
```

Comment:
Suggestion (💡): Maybe it is time we add tests for this `onCleared()` function, wdyt?

Reply:
Absolutely can add them :)

Reply:
in 176d9a0